src/js/common/browserInfoHelper.js

```diff
@@ -169,3 +169,11 @@ export const getBrowserDoNotTrackStatus = () => {
       windowVar.external.msTrackingProtectionEnabled());
   return isBrowserDoNotTrack ? "true" : "false";
 };
+
+/**
+ * This function returns user agent by checking the navigator
+ * @returns {String} user agent for the current browser
+ */
+export const getUserAgent = () => {
+  return navigator.userAgent
```

Comment:
semicolon at the end to be consistent with the style we are using in this repo.
```suggestion
  return navigator.userAgent;
```